schemup/dbs/postgres.py

```diff
@@ -1,11 +1,14 @@
+import six
+
```

Comment:
If we plan on supporting python 2 and 3 (thus the `import six`), it's better to import print_function as it's NOT natively supported by python 2. The following line should do it:

`from __future__ import print_function`